docs/get_started/centos_setup.md

```diff
@@ -0,0 +1,164 @@
+	sudo yum groupinstall -y "Development Tools"
+	sudo yum install -y python27 python27-setuptools python27-tools python-pip
+	sudo yum install -y python27-numpy 
+
+	# install graphviz, jupyter
+	sudo pip install graphviz
+	sudo pip install jupyter
+```
+### Install OpenBLAS
+Note that OpenBLAS can be replaced by other BLAS libs, e.g, Intel MKL.
+
+```bash
+	# Install OpenBLAS at /usr/local/openblas
+	git clone https://github.com/xianyi/OpenBLAS
+	cd OpenBLAS
+	make -j $(($(nproc) + 1))
+	sudo make PREFIX=/usr/local install
+	cd ..
+```
+### Install CUDA for GPU
+If you plan to build with GPU, you need to set up the environment for CUDA and CUDNN.
+
+First, download and install [CUDA 8 toolkit](https://developer.nvidia.com/cuda-toolkit).
+
+Then download [cudnn 5](https://developer.nvidia.com/cudnn).
+
+Unzip the file and change to the cudnn root directory. Move the header and libraries to your local CUDA Toolkit folder:
+
+```bash
+    tar xvzf cudnn-8.0-linux-x64-v5.1-ga.tgz
+    sudo cp -P cuda/include/cudnn.h /usr/local/cuda/include
+    sudo cp -P cuda/lib64/libcudnn* /usr/local/cuda/lib64
+    sudo chmod a+r /usr/local/cuda/include/cudnn.h /usr/local/cuda/lib64/libcudnn*
+    sudo ldconfig
+```
+### Install opencv for Computer Vision and Image Augmentation
+The Open Source Computer Vision (OpenCV) library contains programming functions for computer vision and image augmentation. For more information, see [OpenCV](https://en.wikipedia.org/wiki/OpenCV).
+
+```bash
+	sudo yum install -y cmake
+	# Install OpenCV at /usr/local/opencv
+	git clone https://github.com/opencv/opencv
+	cd opencv
+	mkdir -p build
+	cd build
+	cmake -D BUILD_opencv_gpu=OFF -D WITH_EIGEN=ON -D WITH_TBB=ON -D WITH_CUDA=OFF -D WITH_1394=OFF -D CMAKE_BUILD_TYPE=RELEASE -D CMAKE_INSTALL_PREFIX=/usr/local ..
+	sudo make PREFIX=/usr/local install
+```
+
+## Install MXNet
+
+### Generate libmxnet.so
+After installing the dependencies, use the following command to pull the MXNet source code from GitHub.
+
+```bash
+    # Get MXNet source code
+    git clone https://github.com/dmlc/mxnet.git ~/mxnet --recursive
+    # Move to source code parent directory
+    cd ~/mxnet
+    cp make/config.mk .
+    # Repalce this line if you use other BLAS libs
+    echo "USE_BLAS=openblas" >>config.mk
+    echo "ADD_CFLAGS += -I/usr/include/openblas" >>config.mk
+    echo "ADD_LDFLAGS += -lopencv_core -lopencv_imgproc -lopencv_imgcodecs" >>config.mk
+```
+
+If building with ```GPU``` support, run below commands to add GPU dependency configurations to `config.mk` file:
+
+```bash
+    echo "USE_CUDA=1" >>config.mk
+    echo "USE_CUDA_PATH=/usr/local/cuda" >>config.mk
+    echo "USE_CUDNN=1" >>config.mk
+```
+
+Then build mxnet:
+
+```bash
+    make -j$(nproc)
+```
+
+Executing these commands creates a library called ```libmxnet.so``` in `mxnet/lib/`.
+
+### Install MXNet for Python
+Assume you are in `mxnet` directory , then install Python support for MXNet.
```

Comment:
Next, we install Python interface for MXNet. Assuming you are in "~/mxnet" directory, run below commands.